config/stage.json

```diff
@@ -30,6 +30,7 @@
 		"current-site/domain-warning": true,
 		"current-site/notice": true,
 		"current-site/stale-cart-notice": false,
+		"design-picker/query-marketplace-themes": true,
```

Comment:
I don't think we should enable this flag in staging. In theory, staging and production should behave the same

Reply:
Good point @mmtr, I've raised https://github.com/Automattic/wp-calypso/pull/80902 to set it to false on staging.

Reply:
Thanks @gcsecsey for handling this while I was out 👬